services/web/server/src/simcore_service_webserver/exporter/settings.py

```diff
@@ -1,25 +1,14 @@
 from aiohttp.web import Application
-from pydantic import Field, PositiveInt
+from pydantic import Field
 from servicelib.aiohttp.application_keys import APP_SETTINGS_KEY
 from settings_library.base import BaseCustomSettings
 
 
 class ExporterSettings(BaseCustomSettings):
-    EXPORTER_MAX_UPLOAD_FILE_SIZE: PositiveInt = Field(
-        10,
-        description="size in GB of the maximum projects/import request",
-        env=[
-            "EXPORTER_MAX_UPLOAD_FILE_SIZE",
-            "WEBSERVER_EXPORTER_MAX_UPLOAD_FILE_SIZE",
-        ],
-    )
-    EXPORTER_DOWNLOADER_MAX_TIMEOUT_SECONDS: PositiveInt = Field(
-        60 * 60,
-        description="maximum timeout for each individual file used in the file_downlaoder",
-        env=[
-            "EXPORTER_DOWNLOADER_MAX_TIMEOUT_SECONDS",
-            "WEBSERVER_EXPORTER_DOWNLOADER_MAX_TIMEOUT_SECONDS",
-        ],
+    # NOTE: class will be renamed to `SDSSettings` in the next refactoring
+    # that's why the env var is indexed with `SDS_`
+    SDS_ENABLED: bool = Field(
```

Comment:
WARNING: all envs must start with `EXPORTER_`

Reply:
yes, it is important to keep env var names consistent.
You can do it in next PR but please do not forget about it